packages/form/tests/integration/components/cf-field/input-test.js

```diff
@@ -147,6 +209,62 @@ module("Integration | Component | cf-field/input", function (hooks) {
     assert.dom("input[type=checkbox][value='option-1']").isChecked();
   });
 
+  test("it renders options of a checkbox field vertically", async function (assert) {
+    assert.expect(1);
+
+    await render(hbs`
+      {{cf-field/input
+        field=(hash
+          options=(array
+            (hash slug="option-1" label="Option 1")
+            (hash slug="option-2" label="Option 2")
+          )
+          question=(hash
+            raw=(hash
+              __typename="MultipleChoiceQuestion"
+              meta=(hash
+                vertical=true
+              )
+            )
+          )
+          answer=(hash
+              value=(array "option-1")
+          )
+        )
+      }}
+    `);
+
+    assert.dom(".uk-flex").exists();
+  });
+
+  test("it renders options of a checkbox field horizontally", async function (assert) {
+    assert.expect(1);
+
+    await render(hbs`
+      {{cf-field/input
+        field=(hash
+          options=(array
+            (hash slug="option-1" label="Option 1")
+            (hash slug="option-2" label="Option 2")
+          )
+          question=(hash
+            raw=(hash
+              __typename="MultipleChoiceQuestion"
+              meta=(hash
+                vertical=false
+              )
+            )
+          )
+          answer=(hash
+            value=(array "option-1")
+          )
+        )
+      }}
+    `);
+
+    assert.dom(".uk-flex").doesNotExist();
+  });
+
```

Comment:
Same here. Also, just write a test for the vertical option, not whether it renders normally. This is cluttering the tests.

Reply:
I removed the horizontal options tests.